app/docker/components/datatables/networks-datatable/networksDatatable.js

```diff
@@ -10,6 +10,7 @@ angular.module('portainer.docker').component('networksDatatable', {
     reverseOrder: '<',
     showOwnershipColumn: '<',
     showHostColumn: '<',
-    removeAction: '<'
+    removeAction: '<',
+    endpointStatus: '<'
```

Comment:
Simply pass the newly created binding `offlineMode` here instead of passing the status.